buildbot/dependency.py

```diff
@@ -104,7 +105,8 @@ def main():
 
     print("args:{}".format(args))
 
-    return do_dependency(args)
+    # All dependencies are downloaded and configured as part of the build
+    return True
```

Comment:
I thought CI uses this script to download and build these projects separately.
If you are about to change this workflow, please, remove this file as this change makes it useless.

Reply:
restored old behavior as there are multiple dependencies on this script in CI